.drone.yml

```diff
@@ -28,14 +33,19 @@ build:
     when:
       event: push
       branch: [master, 2.x]
-cache:
-  mount:
-    - /drone/cache
-notify:
-  email:
-    from: noreply@drone.geirsson.com
-    host: smtp.mailgun.org
-    username: noreply@drone.geirsson.com
-    password: $MAILGUN_PASSWORD
-    recipients:
-      - olafurpg+scalameta-ci@gmail.com
```

Comment:
Yes, this should be much easier on drone v0.5, I opened #744 to enable email notifications on build failure.